conf/provider-connector.config/addresses.json

```diff
@@ -3,8 +3,8 @@
   "name": "provider.edc.think-it.io",
   "addresses": {
     "default": "http://localhost:29191/api",
-    "data": "http://localhost:29193/api/v1/data",
-    "ids": "http://provider-connector:9194/api/v1/ids",
+    "data": "http://localhost:19193/management",
+    "ids": "http://consumer-connector:9194/protocol",
```

Comment:
@NabilHouidi why did we need to make this change?

Reply:
since v0.1.0, the default addresses of the connector changed.
there's a typo here. line 7 should be `provider-connector`. will fix